CMakeLists.txt

```diff
@@ -301,7 +301,7 @@ if(BUILD_WITH_ADIOS2)
   target_compile_definitions(mamico INTERFACE BUILD_WITH_ADIOS2)
 endif()
 target_compile_definitions(mamico INTERFACE ${MD_SIM} MDDim3)
-target_compile_options(mamico INTERFACE -Wall -Wno-unknown-warning-option -Wno-unknown-pragmas -Wfatal-errors)
+target_compile_options(mamico INTERFACE -Wall -Wno-unknown-warning-option -Wno-unknown-pragmas -Wfatal-errors -Wno-reorder)
```

Comment:
The problem with this:
This is needed for AutoPas to compile, and I have fixed the issue in the AutoPas master branch. However ls1 does not use the latest AutoPas, it has to be updated to use the latest AutoPas, and the version used by ls1 is pretty outdated. Until ls1 is updated to use the newer versions, we need this flag to compile AutoPas. Do we keep the flag?

Reply:
We keep the flag, but only if `if("${MD_SIM}" STREQUAL "LS1_MARDYN")`